specs/mediawiki/v1/content.yaml

```diff
@@ -910,6 +915,7 @@ paths:
         body:
           wikitext: '{wikitext}'
           bodyOnly: '{bodyOnly}'
+          body_only: '{body_only}'
```

Comment:
This is an example where it'd be nice to have expressions in the templating mechanism.

```yaml
body_only: '{body_only || bodyOnly}'
```